### ISSUE TITLE
Docs for Developer Adding USGSCSM to Dev Environment.

### ISSUE THREAD

**Issue by amystamile-usgs** [DOI-USGS/asc-public-docs#34]
For the average user, when they install ISIS, ISISROOT should be the same as CONDA_PREFIX so usgscsm >= 2.0 will work automatically for users but not for devs.

Steps to installing usgscsm for dev environment:

1. run `conda install conda-forge::usgscsm`

2. Update IsisPreference file under `Group = Plugins` to include `"$CONDA_PREFIX/lib/csmplugins/", -`

Example:

```
Group = Plugins
  CSMDirectory = ("$CONDA_PREFIX/lib/csmplugins/", -
                  "$ISISROOT/lib/csmplugins/", -
                  "$ISISROOT/lib/isis/csm3.0.3/", -
                  "$ISISROOT/csmlibs/3.0.3/", -
                  "$HOME/.Isis/csm3.0.3/")
EndGroup
```

**Comment by acpaquette** [comment]
It may be worth noting, if you are doing USGSCSM development work it would be easier for developers to point to a locally built version of USGSCSM rather than installing it via conda.

Whoever takes on this issue should likely mention/outline how to do the above

**Comment by AustinSanders** [comment]
Closed via #37
